module-3/app/service/mythicalMysfitsService.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+    "os"
+
+    L "../client"
```

Comment:
Relative path imports not recommended. Use full package imports instead. Let me know if you need help with this in your branch as there's a bit of a trick to it. Consider using a more descriptive alias for the client than `L`

Reply:
I'm not sure how to change this to use a full package import.